src/main/java/org/desarrolladorslp/technovation/services/impl/BatchServiceImpl.java

```diff
@@ -74,7 +78,37 @@ public void registerUserToBatch(UUID batchId, UUID userId) {
                     throw new UserAlreadyRegisteredInBatch(user.getId() + " has been registered already");
                 }, () ->
                         batchRepository.registerUserToBatch(batchId, userId));
+    }
+
+    @Override
+    @Transactional
+    public void registerToBatch(List<UUID> usersToRegister, UUID batchId) {
+        List<UUID> errorToRegister = new ArrayList<>();
+        usersToRegister.forEach(
+                uuid -> {
+                    userRepository.findById(uuid).ifPresentOrElse(
+                            user -> {
+                                userRepository.getUserByBatch(batchId,user.getId()).ifPresentOrElse(
+                                    u -> errorToRegister.add(u.getId()), () -> batchRepository.registerUserToBatch(batchId,uuid));
+                            }, () -> errorToRegister.add(uuid)
+                    );
+                });
+        System.out.println("Error al registrar los siguientes teckers " + errorToRegister.stream().collect(Collectors.toList()));
```

Comment:
Use slf4j instead... standard output should never be used to write logs. 

Log the statement at warning level only if errorToRegister is not empty. 

Log at DEBUG level at the begining of the method the message: "Attempt to register <Number of users> users to batch <batchId>"

Log at INFO level at the end of the method: "<Number of users> registered to the batch <batchId>"

Reply:
Logging statements must be written in english